server/modules/surveyRdb/repository/nodeAnalysisTableRepository.js

```diff
@@ -20,3 +21,11 @@ export const createNodeAnalysisTable = async (surveyId, client) => {
     )
   `)
 }
+
+// ===== GRANT PRIVILEGES
+export const grantWriteOnNodeAnalysisToUser = async (surveyId, userName, client) =>
```

Comment:
should we call this name grantWriteToAnalysisUser?

Reply:
ok